app/components/SidebarLayout.tsx

```diff
@@ -13,6 +13,7 @@ import {
   FaLocationDot,
   FaAngleRight,
   FaUserGroup,
+  FaCirclePlus,
```

Comment:
I don't like this icon and am looking forward to polishing the side nav sometime in the near future...